test/ScaleTest.ipynb

```diff
@@ -0,0 +1,1322 @@
+{
```

Comment:
Can you add unit tests for:
1. Scaling one dimension by a factor of `0` (expected to not appear)
2. Scaling one dimension by a negative number. You'll probably need to translate to the middle of the screen first to see where the scaled result lands (expected to treat right as left or up as down and vice versa)

Reply:
PS I like your unit tests for testing scaling by 1 and scaling only one dimension by 1 and another by 3 :+1: